switch/src/main.cpp

```diff
@@ -20,7 +20,7 @@ bool appletMainLoop()
 }
 #endif
 
-#ifndef CHIAKI_SWITCH_ENABLE_LINUX
+#if 0
```

Comment:
the NXLink tool is currently the only way to retrieve logs from the nintendo switch.
Having it enable by default helps "advanced" users to share their logs.